smsdk/client_v0.py

```diff
@@ -317,12 +317,22 @@ def inner(
                 only_names = schema["name"].tolist()[:50]
                 kwargs["_only"] = only_names
```

Comment:
Is it possible to not use the code paths in the `client_v0.py` file?  The code paths in that file are for the old V0 API which we are trying to obsolete.

Reply:
While this suggestion may not be directly related to the reported issue, implementing it may require relocating code from 'client_v0.py' to 'client.py.' This require additional efforts and testing. I would like to track this in a separate Jira item.